examples/webhook.json

```
@@ -191,5 +191,30 @@
   },
   "remoteControl": {
     "result": false
+  },
+  "velocity": {
+    "distinctIp": {
+      "intervals": {
+        "5m": 1,
+        "1h": 1,
+        "24h": 1
+      }
+    },
+    "distinctLinkedId": {
+    },
```

This seems mis-formatted 

```suggestion
    "distinctLinkedId": {}
```

---

pushed